packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/DrawingTask/DrawingTask.js

```diff
@@ -1,56 +1,100 @@
-import { Markdownz } from '@zooniverse/react-components'
 import { Text } from 'grommet'
+import { Blank } from 'grommet-icons'
 import { observable } from 'mobx'
 import { inject, observer, PropTypes as MobXPropTypes } from 'mobx-react'
-import React from 'react'
 import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { Markdownz } from '@zooniverse/react-components'
+import getIcon from './helpers/getIcon'
+import InputIcon from '../InputIcon'
+import InputStatus from '../InputStatus'
+import TaskInputField from '../TaskInputField'
+
+export const StyledFieldset = styled.fieldset`
+  border: none;
+  margin: 0;
+  padding: 0;
+`
+
+const ToolIcon = ({ type }) => {
+  const Icon = getIcon(type)
+  return (
+    <Blank viewBox='0 0 100 100'>
+      <Icon />
+    </Blank>
+  )
+}
+
+const StyledText = styled(Text)`
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  > *:first-child {
+    margin-top: 0;
+  }
+`
 
 function storeMapper (stores) {
-  const {
-    addAnnotation
-  } = stores.classifierStore.classifications
-  const annotations = stores.classifierStore.classifications.currentAnnotations
+  const { active, setActive } = stores.classifierStore.drawing
   return {
-    addAnnotation,
-    annotations
+    active,
+    setActive
   }
 }
 
 @inject(storeMapper)
 @observer
 class DrawingTask extends React.Component {
   onChange (index, event) {
-    const { addAnnotation, task } = this.props
-    if (event.target.checked) addAnnotation(index, task)
+    const { setActive } = this.props
+    if (event.target.checked) {
+      setActive(index)
+    }
   }
 
   render () {
     const {
-      annotations,
+      active,
       task
     } = this.props
-    // let annotation
-    // if (annotations && annotations.size > 0) {
-    //   annotation = annotations.get(task.taskKey)
-    // }
     return (
-      <Text size='small' tag='legend'>
-        <Markdownz>
-          {task.instruction}
-        </Markdownz>
-      </Text>
+      <StyledFieldset>
```

Comment:
I don't think Fieldset is the right element to use here. Fieldsets have simple, inline text legends but our task questions are block-level elements, sometimes with more than one paragraph.

I think this should just be a Grommet Box.